packages/protocol/contracts/dapis/aggregators/Aggregator.sol

```diff
@@ -0,0 +1,17 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.8.2;
+
+/// @title The abstract contract that generalizes aggregation methods
+/// @notice Aggregation method implementations should inherit this contract
+abstract contract Aggregator {
```

Comment:
I would much rather `Reducer`/`Reduce` instead of `Aggregator`/`Aggregate`
https://en.wikipedia.org/wiki/Reduction_Operator
WDYT

Reply:
I agree